### ISSUE TITLE
No audio sometimes when using windowed fullscreen

### ISSUE THREAD

**Issue by Dime333** [Cxbx-Reloaded/Cxbx-Reloaded#1397]
If I press Alt+Enter immediately after starting emulation, there is no audio at all about half of the times. If I wait until the game has started booting and then press Alt+Enter, I don't have this issue. I made a script that sends Alt+Enter when loading games in my frontend and sometimes this happens and sometimes it doesn't. So it seems to be pretty random.

**Comment by Dime333** [comment]
No. That's not it. This also happens when running games without the frontend. If I simply choose an xbe file to run. Press F5 and then Alt+Enter before any intro video or whatever has started, there is often no sound. I just tested Turok 5 times, and I got audio 3 of those times. Even if it was loosing focus, shouldn't the emu produce sounds when it gets the focus back? This should be really easy for you guys to reproduce. By the way, Thank you very much for your awesome work with this :)
